api/api/views/viewsets/mission.py

```diff
@@ -366,6 +371,12 @@ def create(self, request, *args, **kwargs):
                 return Response({
                     'error': 'please specify team',
                 }, status=HTTP_400_BAD_REQUEST)
+        if 'logo' in request.data:
```

Comment:
add a clause to check if you're also in a test:

```python
if 'logo' in request.data and '1' not in ((os.environ.get('TEST, '0'), os.environ.get('CI', '0')):
```

because it will break later